### PR TITLE
wavpack: add support for wavpack DSD

### PR DESCRIPTION
**Pull request by drizzt**
WavPack 5 supports DSD and with DSD the sample rate must be multiplied per 4

### CODE REVIEW

File: mutagen/wavpack.py [hunk 1]
```diff
@@ -91,6 +91,10 @@ def __init__(self, fileobj):
         self.channels = bool(header.flags & 4) or 2
         self.sample_rate = RATES[(header.flags >> 23) & 0xF]
 
+        # most common multiplier (DSD64)
+        if (header.flags >> 31) & 0xF:
```
Comment (lazka) [inline mutagen/wavpack.py:95]: why the `& 0xF` ?

**Comment by lazka** [comment]
Can you add a test? Here is a sample file: 
[dsd.zip](https://github.com/quodlibet/mutagen/files/4568675/dsd.zip)

**Comment by drizzt** [comment]
It should be ok now, with all the requested changes

**Comment by lazka** [comment]
Perfect, thanks!

**Pull request merged**